### PR TITLE
Remove a non-functional procedure.

### PR DESCRIPTION
**Pull request by yhontyk**
> Read our [Contribution guide](https://github.com/eclipse/che-docs/blob/master/CONTRIBUTING.adoc) before submitting a PR.

### PR Checklist

As the author of this Pull Request I made sure that:

- [x] `vale` has been run successfully against the PR branch
- [x] Link checker has been run successfully against the PR branch
- [x] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [x] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [x] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Comment by themr0c** [comment]
It would leave us with no documentation at all about how to include a kubernetes application. Is the feature really removed, or can we request some help to have some content that works?  /cc @l0rd

**Comment by yhontyk** [comment]
@themr0c there will be a working procedure donwstream. I tried to write the upstream one agnostically (without specific examples) but failed. =\ Either both upstream and downstream should be working, or both should be generic.

**Review by themr0c — APPROVED** [review]

**Comment by l0rd** [comment]
Is there an issue for that (upstream or downstream)?

**Comment by yhontyk** [comment]
> Is there an issue for that (upstream or downstream)?

The issue is here https://issues.redhat.com/browse/RHDEVDOCS-2006
Currently, the procedure works for downstream only (well, with the correct examples that are going to be introduced to the donwstream once this PR is merged and downstreamed). I've created a separate issue to fix it in upstream https://issues.redhat.com/browse/RHDEVDOCS-2692 
We will need to either rewrite it without any specific examples of the apps or find working examples that would work on k8s.

File: modules/end-user-guide/partials/assembly_importing-kubernetes-applications-into-a-workspace.adoc [hunk 1]
```diff
@@ -31,7 +31,7 @@ To deploy a new instance of this application in a {prod-short} workspace, use on
 * Modifying an existing workspace: link:#adding-a-kubernetes-application-to-an-existing-workspace-using-the-dashboard_{context}[Using the Dashboard user interface]
 * From a running application: link:#generating-a-devfile-from-an-existing-kubernetes-application_{context}[Generating a devfile with `{prod-cli}`]
 
-include::partial$proc_including-kubernetes-applications-in-a-workspace-devfile-definition.adoc[leveloffset=+1]
+include::partial$proc_{prod-id-short}-including-kubernetes-applications-in-a-workspace-devfile-definition.adoc[leveloffset=+1]
```
Comment (themr0c) [inline modules/end-user-guide/partials/assembly_importing-kubernetes-applications-into-a-workspace.adoc]: Context specific files should go in the examples directory.

Comment (themr0c) [reply modules/end-user-guide/partials/assembly_importing-kubernetes-applications-into-a-workspace.adoc]:
```suggestion
include::example$proc_{prod-id-short}-including-kubernetes-applications-in-a-workspace-devfile-definition.adoc[leveloffset=+1]
```
+ move the file to the examples directory.

File: modules/end-user-guide/partials/proc_che-including-kubernetes-applications-in-a-workspace-devfile-definition.adoc [hunk 2]
```diff
@@ -34,47 +34,13 @@ metadata:
...
- - type: kubernetes
-   reference: https://raw.githubusercontent.com/.../mongo-db.yaml
- - alias: nodejs-app
-   type: kubernetes
-   reference: https://raw.githubusercontent.com/.../nodejs-app.yaml
-   entrypoints:
-     - command: ['sleep']  <1>
-       args: ['infinity']
-----
+include::example$snip_including-kubernetes-applications-in-a-workspace-devfile-definition-nodejs-minimal.adoc[]
```
Comment (themr0c) [inline modules/end-user-guide/partials/proc_che-including-kubernetes-applications-in-a-workspace-devfile-definition.adoc:37]: If the examples are not context specific, there is no need to put them in a separate file.

**Review by l0rd — APPROVED** [review]

**Pull request merged**